.babelrc

```diff
@@ -8,6 +8,9 @@
     "lodash"
   ],
   "env": {
+    "browser": {
+      "plugins": ["add-module-exports"]
+    },
```

Comment:
We need module exports for the window object, prevents us from having to do `stardust.default.Button`.